mem0/llms/gemini.py

```diff
@@ -138,9 +137,9 @@ def generate_response(
             "top_p": self.config.top_p,
         }
 
-        if response_format:
+        if response_format is not None and response_format == "json_object":
```

Comment:
This code makes Gemini callable and returns correctly (no schema defined yet, it will rely on prompts)